packages/edit-site/src/components/site-hub/index.js

```diff
@@ -186,8 +181,7 @@ export const SiteHubMobile = memo(
 							className="edit-site-site-hub__actions"
 						>
 							<Button
-								// TODO: Switch to `true` (40px size) if possible
-								__next40pxDefaultSize={ false }
+								__next40pxDefaultSize
```

Comment:
This can be seen when you visit and check for search icon on the site editor mode on mobile.

| Before | After     |
| :-------- | :------- |
| <img width="419" alt="site-hub-mobile-command-before" src="https://github.com/user-attachments/assets/9a753856-76b2-4eb6-869b-44be313d23b8"> | <img width="419" alt="site-hub-mobile-command-after" src="https://github.com/user-attachments/assets/74510c88-0058-442a-a41f-f68640293dc5"> |